scripts/check_ecosystem.py

```diff
@@ -31,13 +32,12 @@ class Repository(NamedTuple):
 
     org: str
     repo: str
-    ref: Optional[str]
+    ref: str | None
     select: str = ""
     ignore: str = ""
     exclude: str = ""
 
-    @asynccontextmanager
```

Comment:
It looks like the usage was removed here?

Reply:
Thanks, fixed the bad rebase.